src/Contract.php

```diff
@@ -22,8 +29,35 @@ class Contract extends Model
      */
     public function __construct(array $attributes = [])
     {
-        $this->fillables('file', 'status', 'signed_at', 'options');
-        $this->columns('file', 'status', 'signed_at', 'options');
+        $this->fillables('file', 'status', 'signed_at', 'options', 'template', 'external_id', 'user_id');
+        $this->columns('file', 'status', 'signed_at', 'options', 'external_id', 'user_id');
         parent::__construct($attributes);
     }
+
+    public function toPdf(HtmlConverterInterface $htmlConverter)
+    {
+        if (!$this->template) {
+            return;
+        }
+
+        $page = view($this->template);
+        $html = $page->render();
+        $path = 'contracts/contract.pdf';
+        $htmlConverter->page($html)->save($path);
+        $this->file = $path;
+        $this->unsetAttribute('template');
+    }
+
+    public function sign(DocumentSignInterface $documentSign)
+    {
+        if (!$this->file) {
```

Comment:
add a config to mvc-contracts: document-sign.enabled: true/false (filled via .env)
check here in the if statement for the enabling of the service

Reply:
There is no mvc-contract.. Don't you mean mvc-document-sign?